src/components/ProgressIndicator/_progress-indicator.scss

```diff
@@ -1,82 +1,306 @@
 @import '~carbon-components/scss/components/progress-indicator/progress-indicator';
 @import '../../globals/vars';
 
+$IBMPlexSans: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
+
 .#{$iot-prefix}--progress-indicator {
-  &.#{$prefix}--progress--vertical {
-    .#{$prefix}--progress-step {
-      overflow: hidden;
+  display: flex;
+  overflow: hidden;
+
+  .label-container {
+    position: absolute;
+    left: 0;
+    top: $spacing-07;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: inherit;
+    height: 100%;
+    display: block;
+  }
+
+  .label {
+    position: relative;
+    font-family: $IBMPlexSans;
+    font-size: 14px;
+    color: #161616;
+    letter-spacing: 0.16px;
+    line-height: 18px;
```

Comment:
the site is down so I can't confirm but there may be a carbon text mixin that we should be using here.